demos/k8s-demo/chart/demo-app-secretless/README.md

```diff
@@ -0,0 +1,152 @@
+# demo-app-secretless
+
+[The Secretless Broker](https://github.com/cyberark/secretless-broker) is a connection broker which relieves client applications of the need to directly handle secrets to target services such as databases, web services, SSH connections, or any other TCP-based service.
+
+## TL;DR;
+
+```bash
+$ helm install -f values.yaml .
+```
+
+## Introduction
+
+This chart bootstraps a deployment of a configurable demo-application, which uses [PostgreSQL](https://github.com/docker-library/postgres) as its backend, with Secretless on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. Secretless seamlessly manages the database connection for this application.
```

Comment:
it's hard to understand what this sentence is saying. 

maybe for clarity it could be revised to:
> This chart uses the Helm package manager to bootstrap a Kubernetes deployment of an application with a PostgreSQL backend together with a Secretless sidecar.